dvc/command/ls.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import logging
+
+from dvc.command.base import append_doc_link
+from dvc.command.base import CmdBaseNoRepo
+from dvc.exceptions import DvcException
+
+
+logger = logging.getLogger(__name__)
+
+
+class CmdList(CmdBaseNoRepo):
+    def run(self):
+        from dvc.repo import Repo
+
+        try:
+            outs = Repo.ls(
+                self.args.url,
+                self.args.target,
+                rev=self.args.rev,
+                recursive=self.args.recursive,
+                outputs_only=self.args.outputs_only,
+            )
+            logger.info("\n".join(map(str, outs)))
+            return 0
+        except DvcException:
+            logger.exception("failed to list '{}'".format(self.args.url))
+            return 1
+
+
+def add_parser(subparsers, parent_parser):
+    LIST_HELP = "List files and DVC outputs in the repo"
+    list_parser = subparsers.add_parser(
+        "list",
+        parents=[parent_parser],
+        description=append_doc_link(LIST_HELP, "list"),
+        help=LIST_HELP,
+        formatter_class=argparse.RawTextHelpFormatter,
+    )
+    list_parser.add_argument(
+        "url",
+        help="Supported urls:\n"
+        "/path/to/file\n"
+        "/path/to/directory\n"
+        "C:\\\\path\\to\\file\n"
+        "C:\\\\path\\to\\directory\n"
+        "https://github.com/path/to/repo\n"
+        "git@github.com:path/to/repo.git\n",
+    )
+    list_parser.add_argument(
+        "-R", "--recursive", action="store_true", help="Recursively list files"
+    )
+    list_parser.add_argument(
+        "--outputs-only", action="store_true", help="Show only dvc outputs."
```

Comment:
We have a similar option called `--outs-only` in `dvc remove`, which is admitedly not our most popular command, but I would consider keeping it consistent. CC @jorgeorpinel

Reply:
Agree. I vote for `--outs-only`.

Reply:
@JIoJIaJIu please also capitalize DVC in the `help` string.
Oh and please end the previous option's `help` in period `.` as well.
Thanks

Reply:
Sure, @jorgeorpinel, thanks for pointing me out